help/sources/connectors/marketing-automation/pathfactory.md

```diff
@@ -0,0 +1,68 @@
+---
+title: PathFacory Source Overview
+description: Learn how to connect PathFactory to Adobe Experience Platform using APIs or the user interface.
+last-substantial-update: 2024-04-20
+badge: Beta
+exl-id: 
+---
+# [!DNL PathFactory]
+
+>[!NOTE]
+>
+>The [!DNL PathFactory] source is in beta. Please read the [sources overview](../../home.md#terms-and-conditions) for more information on using beta-labeled sources.
+
+Adobe Experience Platform allows data to be ingested from external sources while providing you with the ability to structure, label, and enhance incoming data using Platform services. You can ingest data from a variety of sources such as Adobe applications, cloud-based storage, databases, and many others.
+
+Experience Platform provides support for ingesting data third-party marketing automation system. Support for marketing automation providers includes [!DNL PathFactory].
+
+[[!DNL PathFactory]](https://www.pathfactory.com/) offers a cloud-based platform that helps businesses manage content journeys and drive engagement through intelligent content insights. This guide details how you can integrate data from PathFactory into your Experience Platform, utilizing PathFactory's connectors for optimal data ingestion.
+
+You can ingest data from [[!DNL PathFactory]](https://www.pathfactory.com/) using three primary sources:
+
+1. [!DNL Visitors]: Ingest customer and contact data as records to better understand your audience.
+2. [!DNL Sessions]: Time series events that track individual user session activities on your platform.
+3. [!DNL Page Views]: Time series events that provide insights into which pages are being viewed, helping you analyze content performance.
+
+## IP address allow list {#ip-allow-list}
+
+A list of IP addresses may require to be added to an allow list prior to working with source connectors. Failing to add your region-specific IP addresses to your allow list may lead to errors or non-performance when using sources. See the [IP address allow list](../../ip-address-allow-list.md) page for more information.
+
+## Prerequisites {#prerequisites}
+
+Before you begin integrating [[!DNL PathFactory]](https://www.pathfactory.com/) connectors with your Experience Platform, ensure you meet the following prerequisites:
```

Comment:
Revise to: "...connectors with Experience Platform, you must ensure that you meet the following prerequisites:"

Reply:
Fixed